OpenSim/Simulation/Manager/Manager.cpp

```diff
@@ -822,11 +822,18 @@ void Manager::initialize(const SimTK::State& s)
         _timeStepper->initialize(s);
         _timeStepper->setReportAllSignificantStates(true);
     }
+
+    // Here we call the constructStorage because it is possible that the Model's
+    // control storage has already be appended in a previous simulation [Dimitar
+    // Stanev; see Manager memory leak PR].
```

Comment:
All looks good. The only thing I'd maybe change is to change this comment to refer to a URL or issue number just so that it's easy for a later maintenance dev to pull up what the original issue was (if they're wondering)